### PR TITLE
Bump version: 2.32.11 → 2.33.0: Allow launching of non-preemptible jobs

### PR DESCRIPTION
**Pull request by lgruen**

### CODE REVIEW

File: analysis_runner/cli_analysisrunner.py [hunk 1]
```diff
@@ -57,6 +57,14 @@ def add_analysis_runner_args(parser=None) -> argparse.ArgumentParser:
         ),
     )
 
+    parser.add_argument(
+        '--preemptible',
+        required=False,
+        action=argparse.BooleanOptionalAction,
+        default=True,
```
Comment (illusional) [inline analysis_runner/cli_analysisrunner.py:64]: How does default true here work if it's not a flag, `--preemptible false`?

File: server/main.py [hunk 2]
```diff
@@ -153,6 +154,9 @@ async def index(request):
         job.cpu(cpu)
     if memory:
         job.memory(memory)
+    if preemptible is not None:
```
Comment (illusional) [inline server/main.py:157]: Should we default to True here if `None` to be consistent with the CLI description?

**Review by illusional — APPROVED** [review]
Minor comments but looks good!

File: analysis_runner/cli_analysisrunner.py (hunk 1 above)
Comment (lgruen) [reply analysis_runner/cli_analysisrunner.py:64]: As far as I understand, it should understand both `--preemptible` and `--no-preemptible` and set the value accordingly.

File: server/main.py (hunk 2 above)
Comment (lgruen) [reply server/main.py]: Good idea! I've added a default value when fetching the value from the dictionary.

**Pull request merged**